recipes/ccp4srs/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "CCP4SRS" %}
+{% set version = "2024.06.14" %}
+{% set sha256 = "935b2d99b73f5f8b2396a5f2b6cdc617610131b5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://ccp4forge.rc-harwell.ac.uk/ccp4/{{ name|lower }}/-/archive/{{ sha256 }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage(name|lower, max_pin=None) }}
+
+requirements:
+  build:
+    - {{ compiler("cxx") }}
+    - autoconf
+    - automake
+    - gnuconfig
+    - libtool
+    - make
+    - pkg-config
+  host:
+    - mmdb2
+    - zlib
+
+test:
+  commands:
+    - srsgen --version
+    - srsview --version
+    - pdb2to3 --version
+
+about:
+  home: "https://www.ccp4.ac.uk"
+  dev_url: "https://ccp4forge.rc-harwell.ac.uk/ccp4/ccp4srs"
+  doc_url: "https://www.ccp4.ac.uk/html"
+  license: "UNKNOWN"

```

Comment:
Can you contact the authors and ask them what license they intend for this? It's legally problematic to distribute things without a known license.

Reply:
The source code comment says modified LGPL-3.0.

I think it is OK, but since there is no license file, I will send a confirmation email just to be sure.